dietpi/dietpi-autostart

```diff
@@ -90,6 +90,8 @@ _EOF_
 			"3" "RetroPie"
 			"4" "OpenTyrian"
 			"5" "DietPi-Cloudshell"
+			"6" "MATE: Desktop"
+			"7" "GNUStep: Desktop"
```

Comment:
AH!  Thank you!  I had searched and searched for those as so startx, etc would get kicked off.